.github/workflows/ci.yml

```
@@ -3,7 +3,7 @@ name: CI
 on: [push, pull_request]
 
 env:
-  minrust: 1.51.0
+  minrust: 1.57.0
```

```suggestion
  minrust: 1.51.0
```
This can be reverted now that base64 0.21.3 has MSRV 1.47